endpoints/openrtb2/auction_test.go

```diff
@@ -4669,3 +4729,40 @@ func getIntegrationFromRequest(req *openrtb_ext.RequestWrapper) (string, error)
 	reqPrebid := reqExt.GetPrebid()
 	return reqPrebid.Integration, nil
 }
+
+type rejectableHookExecutor struct {
+	entrypointReject,
+	rawAuctionReject,
+	processedAuctionReject *hookexecution.RejectError
```

Comment:
are these left for future tests? because they are not used anywhere

Reply:
yes, these are for other hooks